README.md

```diff
@@ -1049,6 +1083,19 @@ On OS X and Linux, your git configuration file is stored in ```~/.gitconfig```.
     zap = fetch -p
 ```
 
+<a name="adding-empty-repository"></a>
+### I want to add an empty directory to my repository
+
+You can’t! git doesn’t support this, but there’s a hack. You can create a .gitignore file in the directory with the following contents:
+
+
+```sh
+$ # Ignore everything in this directory
```

Comment:
No need for the `$` prompts, there. The file also shouldn't be in a `shell` block, AFAIK.

Reply:
Might be good to add:

Reply:
> Another common convention is to make an empty file in the folder, titled `.gitkeep`. 

```sh
$ mkdir mydir
$ touch mydir/.gitkeep
```